### PR TITLE
Done TODO in test_wrapper.py, scheduler.py, and state.py

### PR DESCRIPTION
**Pull request by hieuchi911**

### CODE REVIEW

**Review by fostiropoulos — APPROVED** [review]

File: ablator/modules/scheduler.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ class PlateuaConfig(SchedulerArgs):
     patience: int = 10
     min_lr: float = 1e-5
     mode: str = "min"
-    factor: float = 0.0 # TODO {fixme} this is error prone -> new_lr = 0
+    factor: float = 0.1
```
Comment (fostiropoulos) [inline ablator/modules/scheduler.py:181]: Shouldn't the documentation also be updated?

File: ablator/modules/scheduler.py [hunk 2]
```diff
@@ -178,7 +178,7 @@ class PlateuaConfig(SchedulerArgs):
...
     threshold: float = 1e-4
     verbose: bool = False
     # TODO fix mypy errors for custom types
```
Comment (fostiropoulos) [inline ablator/modules/scheduler.py:184]: is this addressed?

**Review by fostiropoulos — CHANGES_REQUESTED** [review]

File: ablator/modules/scheduler.py (hunk 2 above)
Comment (hjzccc) [reply ablator/modules/scheduler.py:184]: Yes, all custom types should be fixed by plugin, but here it is ty.Literal from python typing.

**Review by fostiropoulos — APPROVED** [review]

File: ablator/modules/scheduler.py (hunk 2 above)
Comment (fostiropoulos) [reply ablator/modules/scheduler.py:184]: @h21565897 it should throw an error if inheriting from `ty.Literal` can you please investigate.

**Review by fostiropoulos — CHANGES_REQUESTED** [review]
Potential Bug

**Comment by fostiropoulos** [comment]
@h21565897 @hieuchi911 @MrLYG can we please update on the status for this?

**Pull request closed without merging**